mecanum_drive_controller/test/diff_drive_test.cpp

```diff
@@ -0,0 +1,151 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Hello @efernandez, I am away from the office during the next 3 days. But I'll come back to you on Monday with the equations and other answers.